replay/metrics/base_metric.py

```diff
@@ -99,6 +99,23 @@ def preprocess_gt(
     return true_items_by_users
 
 
+def filter_sort(recommendations: AnyDataFrame) -> DataFrame:
```

Comment:
you add new function `filter_sort` but do not remove the old one, `sorter`. where is it used for now (at least NCIS metric)? do we need to completely remove `sorter`? `sorter` allows to return an extra data. need to discuss